docs/integrations/arq.rst

```diff
@@ -16,11 +16,8 @@ How to use
 
 ..  code-block:: python
 
-    from dishka.integrations.arq import (
-        FromDishka,
-        inject,
-        setup_dishka,
-    )
+    from dishka import FromDishka
```

Comment:
for all integrations we have FromDishka within them. Thought it is jsut reexport it can be usuful for future customizations